templates/layout/sidebar.html.twig

```diff
@@ -0,0 +1,94 @@
+<div class="sidebar-sticky">
+    <ul class="nav flex-column">
+        <li class="nav-item">
+            <a class="nav-link {{ app.request.attributes.get('_route') matches '#^(index)$#' ? 'active' }}" href="{{ path('index') }}">
+                <i class="feather fa fa-home"></i>
+                Dashboard
+            </a>
+        </li>
+    </ul>
+
+    <h6 class="sidebar-heading d-flex justify-content-between align-items-center px-3 mt-4 mb-1 text-muted">
+        <span>CRM</span>
+    </h6>
+    <ul class="nav flex-column">
+        <li class="nav-item">
+            <a class="nav-link {{ app.request.attributes.get('_route') matches '#^(speaker)#' ? 'active' }}" href="{{ path('speaker_index') }}">
+                <i class="feather fa fa-chalkboard-teacher"></i>
+                Speakers
+            </a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link {{ app.request.attributes.get('_route') matches '#^(organisation)#' ? 'active' }}" href="{{ path('organisation_list') }}">
+                <i class="feather fa fa-building"></i>
+                Organisations
+            </a>
+        </li>
```

Comment:
You forgot in this the Interview Questions part
All CRM parts have a meaning outside selecting an event. Example: 
Speaker as prospects before they attenpted to go to an event present a talk
Organizations as prospect that did sponsor an old event but could be asked to be sponsor at the new one